crates/sui-adapter-transactional-tests/tests/upgrade/basic.exp

```diff
@@ -0,0 +1,12 @@
+processed 3 tasks
+
+init:
+A: object(0,0)
+
+task 1 'publish'. lines 6-11:
+created: object(1,0), object(1,1)
+mutated: object(0,0)
+
+task 2 'upgrade'. lines 13-17:
+created: object(2,0)
+mutated: object(0,0), object(1,1)
```

Comment:
Right now the test passes when it should fail (a public function is removed in the fixture).